metadata-ingestion/src/datahub/ingestion/source/metadata/business_glossary.py

```diff
@@ -148,22 +149,41 @@ def make_glossary_term_urn(
 
 def get_owners(owners: Owners) -> models.OwnershipClass:
     owners_meta: List[models.OwnerClass] = []
+
+    valid_owner_types = [
+        attr
+        for attr in dir(models.OwnershipTypeClass)
+        if not callable(getattr(models.OwnershipTypeClass, attr))
+        and not attr.startswith("__")
+    ]
+
+    if owners.type is not None and owners.type not in valid_owner_types:
+        raise ValueError(
+            f"Invalid owner type: {owners.type}. Valid owner types are: {', '.join(valid_owner_types)}"
+        )
+
     if owners.users is not None:
-        owners_meta = owners_meta + [
+        owners_meta += [
             models.OwnerClass(
                 owner=make_user_urn(o),
-                type=models.OwnershipTypeClass.DEVELOPER,
+                type=getattr(
+                    models.OwnershipTypeClass, owners.type or "DEVELOPER"
```

Comment:
Is it possible that usage of default value for `ConfigModel` field, like for example [here](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/transformer/add_dataset_ownership.py#L102) could let us avoid writing this logic and repeating it (along with default value)?